Docs/Configuration.tex

```diff
@@ -3653,16 +3651,16 @@ \subsection{Security Properties}\label{miscsecurityprops}
   allowing to enabling Apple Secure Boot with any SMBIOS. Setting
   \texttt{SecureBootModel} to any valid value but \texttt{Disabled} is equivalent to
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Medium Security}}
-  of Apple Secure Boot. To achieve \texttt{Full Security} one will need to
-  also specify \texttt{ApECID} value.
+  of Apple Secure Boot. The \texttt{ApECID} value must also be specified to
+  achieve \texttt{Full Security}.
 
   Enabling Apple Secure Boot is more demanding to incorrect configurations,
-  buggy macOS installations, and unsupported setups. Things to keep in mind:
+  buggy macOS installations, and unsupported setups. Things to consider:
 
   \begin{enumerate}
     \tightlist
-    \item Just like on T2 Macs one will not be able to install any unsigned kernel
-      drivers and several signed kernel drivers including NVIDIA Web Drivers.
+    \item As with T2 Macs, unsigned kernel drivers and several signed kernel
```

Comment:
just *With T2 Macs* maybe?

Reply:
Don't think so. This is saying the behaviour is like it is on T2 Macs.

Reply:
Makes sense. Sorry for confusion.